packages/Microsoft.Bot.Components.Telephony/Schemas/CallTransferDialog.uischema

```diff
@@ -0,0 +1,26 @@
+﻿{
+  "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
+  "form": {
+    "label": "Transfer a call",
+    "subtitle": "Call Transfer",
+    "order": [
+      "targetPhoneNumber",
+      "*"
+    ],
+    "properties": {
+      "targetPhoneNumber": {
+        "intellisenseScopes": [
+          "variable-scopes"
+        ]
+      }
+    }
+  },
+  "menu": {
+    "label": "Transfer a call",
+    "submenu": [ "Channels", "Telephony" ]
```

Comment:
We don't use the word channels in composer, we try to use Connections. I can run this by @sgellock / @benbrown / design on the wording here.

Reply:
We want to make it channels as these actions are Channel specific - discussed this with Steve